scripts/dist.ts

```diff
@@ -1,9 +1,4 @@
-import { Temporal } from "@js-temporal/polyfill";
-import {
-  BASELINE_LOW_TO_HIGH_DURATION,
-  computeBaseline,
-  setLogger,
-} from "compute-baseline";
+import { computeBaseline, setLogger } from "compute-baseline";
```

Comment:
Oh oops, my editor removed these unused imports. I can move this to a separate PR, if desired.